crates/nu-engine/src/evaluate/variables.rs

```diff
@@ -17,18 +21,33 @@ pub fn nu(scope: &Scope, tag: impl Into<Tag>) -> Result<Value, ShellError> {
             dict.insert_untagged(v.0, UntaggedValue::string(v.1));
         }
     }
+
     nu_dict.insert_value("env", dict.into_value());
 
-    let config_file = match scope.get_var("config-path") {
```

Comment:
Please help me understand why are we removing `$nu.config-path`?

Reply:
Running: `echo $nu.config-path` will still give the expected output :)
It's right on the other side:  https://github.com/nushell/nushell/pull/3265/files#diff-c8858d9cba0d39d1870b2e7169ae3c176590b812be484adf52f62c076eb2ac89R38-R41

Reply:
lolz. thanks!